migrations/add-plugin-to-user-ident.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "user" ADD COLUMN plugin varchar;
+UPDATE "user" SET plugin = 'learn';
+ALTER TABLE "user" ALTER COLUMN plugin SET NOT NULL;
+ALTER TABLE "user" DROP CONSTRAINT unique_user;
+ALTER TABLE "user" ADD CONSTRAINT "unique_user" UNIQUE(ident, plugin);
```

Comment:
This is probably fine here, but restarting the app will auto-add this index. It might even drop the existing one first (since the name is the same) but that I'd have to test.